source/options.html

```diff
@@ -9,17 +9,25 @@ <h2>Disable features</h2>
 	<p>
 		List the features to disable, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a>. Enable logging to show in the console what features are enabled on each page.
 	</p>
-	<p class="small">
-		<textarea name="disabledFeatures" rows="3" placeholder="For example: &#10;mark-unread hide-own-stars"></textarea> <br>
-		<strong>Notice:</strong> Mostly untested; only JavaScript-based features can be disabled.
+	<p>
+		<textarea name="disabledFeatures" rows="3" placeholder="For example: &#10;mark-unread hide-own-stars"></textarea>
+		<div class="small">
+			<strong>Notice:</strong> Mostly untested; only JavaScript-based features can be disabled.
+		</div>
 	</p>
 	<p>
 		<label>
 			<input type="checkbox" name="logging">
-			Enable logging
+			<span>Enable logging</span>
```

Comment:
Nor does this. If you want to add spacing you can add `margin-right` to `input[type='checkbox']`